ArduPlane/quadplane.cpp

```diff
@@ -1116,6 +1116,13 @@ void QuadPlane::control_qacro(void)
     }
 }
 
+void QuadPlane::relax_attitude_control(){
+    // if not a vectored tailsitter completely disable attitude control
+    if (!is_tailsitter() || !(tailsitter.vectored_hover_gain > 0)) {
```

Comment:
vectored_hover_gain defaults to 0.5, so the majority of copter tail sitters will still have it enabled. I think its better to look for the tilt servo outputs.

Reply:
right, I missed your comments above regarding tilt servos and relaxing only roll/yaw
I'll implement those changes too